cmd/build/v2/services.go

```diff
@@ -52,7 +54,15 @@ func (bc *OktetoBuilder) GetServicesToBuild(ctx context.Context, manifest *model
 	}
 	close(toBuild)
 
+	// create a spinner to be loaded while checking if images needs to be built
+	s := spinner.New(spinner.CharSets[0], 100*time.Millisecond)
```

Comment:
We have a wrapper for this library. You can use `oktetoLog.Spinner("Building images...")`. 

This will create the same spinner as the other ones across the application
To stop it we call `defer oktetoLog.StopSpinner()` to stop it at the end of the function.

Reply:
Resolved!